rocAL/rocAL/include/loaders/image/node_cifar10_loader.h

```diff
@@ -31,7 +31,7 @@ class Cifar10LoaderNode: public Node
     /// \param device_resources shard count from user
 
     /// internal_shard_count number of loader/decoders are created and each shard is loaded and decoded using separate and independent resources increasing the parallelism and performance.
-    Cifar10LoaderNode(Image *output, void *device_resources);
+    Cifar10LoaderNode(Tensor *output, void *device_resources);
```

Comment:
Why don't we have a set_output function for cifar data loader?

Reply:
set_output is not present in the node_loader files. Please check